lib/pages/chat/chat_pinned_events/pinned_messages.dart

```diff
@@ -129,6 +184,23 @@ class PinnedMessagesController extends State<PinnedMessages>
     );
   }
 
+  void onSelectMessage(Event event) {
+    if (!event.redacted) {
+      if (selectedEvents.contains(event)) {
+        setState(
+          () => selectedEvents.remove(event),
+        );
```

Comment:
Why dont use Notifier?

Reply:
took same logic as chat list, I changed to notifier